tests/php/SiteConfigTest.php

```diff
@@ -23,6 +24,10 @@ public function testCanCreateRootPages()
         /** @var SiteConfig $config */
         $config = $this->objFromFixture(SiteConfig::class, 'default');
 
+        // Admin trumps all
+        $this->logInWithPermission('ADMIN');
```

Comment:
Thanks to https://github.com/silverstripe/silverstripe-framework/pull/7403 instead of logging in and out with users you can now use:

```php
$this->actWithPermission('ADMIN', function () use ($config) {
    $this->assertTrue($config->canCreateTopLevel());
});
```

Reply:
Oh, that's much easier to read! I'll change the tests to use that.